server_manager/model/server.ts

```diff
@@ -39,6 +39,9 @@ export interface Server {
   // Returns stats for bytes transferred across all access keys of this server.
   getDataUsage(): Promise<BytesByAccessKey>;
```

Comment:
Let's merge these methods and have a single `getUsageMetrics()` method instead. It will simplify the logic. We don't want to implement all the stats flow for each new metric.